native-builds/libHarfBuzzSharp_windows/libHarfBuzzSharp.vcxproj

```diff
@@ -185,6 +187,7 @@
     <!--HB_BASE_RAGEL_GENERATED_sources-->
     <ClInclude Include="..\..\externals\harfbuzz\src\hb-buffer-deserialize-json.hh" />
     <ClInclude Include="..\..\externals\harfbuzz\src\hb-buffer-deserialize-text.hh" />
+    <ClInclude Include="..\..\externals\harfbuzz\src\hb-number-parser.hh.hh" />
```

Comment:
Should be `..\..\externals\harfbuzz\src\hb-number-parser.hh`

Reply:
ah, gotcha - let me fix that

Reply:
More and more is green! I think at this point it is just UWP and the use of non-UWP members. But that can be easily fixed (I hope)

Reply:
So I think that was just me including GDI on UWP - so I decided for this major version to also remove GDI from the Windows build - should be a smaller dll now. We got the newer and better DirectWrite.

Reply:
Y should be fine to exclude it from the UWP build. If someone wanted to load GDI fonts one could do that on managed side.

Reply:
To make things easier it is very possible to have a custom font manager that is customizable via callbacks from managed side. So it is possible to avoid these dependencies. The manager for DWrite, GDI, FontConfig, etc could then be written in C#. Sure that has some drawbacks but it would be possible to use SkiaSharp in more restricted environments.

So excited about this major update 👍